pages/spotlight/[slug].tsx

```diff
@@ -44,6 +45,7 @@ const Spotlight: NextPage<SpotlightProps> = ({ setData, scaffold, page }) => {
           countryCode={page.country_code}
           countryName={page.country_name}
         />
+        <LocationComparisonSection countryCode={'UG'} />
```

Comment:
Strange ... with code that's using the country code right in clear view, you opted to hardcode ...

Reply:
Updated